protocol-swanky/src/main/scala/org/ensime/server/protocol/swank/SwankFormats.scala

```diff
@@ -79,6 +79,14 @@ object SwankProtocolCommon {
     }
   }
 
+  implicit object HierarchyInfoFormat extends SexpFormat[HierarchyInfo] {
+    def write(hi: HierarchyInfo): Sexp = hi match {
+      case value: ClassInfo => ???
```

Comment:
Yes, I wasn't sure if those were the right way to do it in the first place so kept it for later.